templates/.env.j2

```diff
@@ -0,0 +1,10 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Change the following values for local development.
+SECRET_KEY= secret-key
```

Comment:
As the name implies, really we ought to treat this as a secret (otherwise users can modify & sign their own session cookies) but I appreciate that for our app this isn't as critical as the Trello secrets